<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetAssignedTaskList</class>
 <widget class="QDialog" name="GetAssignedTaskList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Get Assigned Task List</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>382</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>AssignedTaskList</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TaskID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Task Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Task Description</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTaskID">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>31</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTaskTitle">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTaskDescription">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
